docs/src/config-file.md

```diff
@@ -67,6 +67,10 @@ postgres:
       source_id_format: 'table.{schema}.{table}.{column}'
       # Add more schemas to the ones listed above
       from_schemas: my_other_schema
+      # A table column to use as the feature ID
+      # If a table has no column with this name, `id_column` will not be set for that table.
+      # If a list of strings is given, the first found column will be treated as a feature ID.
+      id_column: feature_id
```

Comment:
This should probably be `id_columns`?  Note that both are allowed because of the Serde's `alias` attribute.